### PR TITLE
{AKS} Fix env setup in test pipelines

### PR DESCRIPTION
**Pull request by FumingZhang**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

Fix env setup in test pipelines.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
AKS

**Review by wangzelin007 — APPROVED** [review]
LGTM

File: src/aks-preview/azcli_aks_live_test/scripts/setup_venv.sh [hunk 1]
```diff
@@ -28,7 +28,7 @@ setupAZ(){
     ext_repo=${2:-""}
 
     # install azdev, used later to install azcli and extension
-    pip install azdev==0.1.32
+    pip install azdev
```
Comment (andyliuliming) [inline src/aks-preview/azcli_aks_live_test/scripts/setup_venv.sh:31]: I think it's better to pin to one version (higher version with the fix).

**Review by andyliuliming — CHANGES_REQUESTED** [review]

File: src/aks-preview/azcli_aks_live_test/scripts/setup_venv.sh (hunk 1 above)
Comment (FumingZhang) [reply src/aks-preview/azcli_aks_live_test/scripts/setup_venv.sh]: Good point, fixed in commit [#8f24632](https://github.com/Azure/azure-cli-extensions/pull/4072/commits/8f24632c4d680dd07fab8a6fc8a0b331decc88c6).

**Review by andyliuliming — APPROVED** [review]

**Review by zhoxing-ms — APPROVED** [review]

**Pull request merged**